src/main/kotlin/no/nav/familie/pdf/pdf/UtilsMetaData.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.familie.pdf.pdf
+
+import com.itextpdf.kernel.pdf.PdfName
+import com.itextpdf.kernel.pdf.PdfString
+import com.itextpdf.kernel.pdf.PdfViewerPreferences
+import com.itextpdf.kernel.xmp.XMPMeta
+import com.itextpdf.kernel.xmp.XMPMetaFactory
+import com.itextpdf.pdfa.PdfADocument
+
+/**
+ * `UtilsMetaData` er en hjelpekklasse for å legge til metadata i et PDF-dokument.
+ * Vi setter metadataene på to ulike måter for å dekke forskjellige behov:
```

Comment:
Lang kommentar? Kanskje det holder med typ "Metadata for skjermleser" over den ene, og "Maskinlesbar metadata" over den andre om vi vil ha med en forklaring:)

Reply:
kan korte ned litt